app/src/main/res/layout/fragment_main.xml

```diff
@@ -55,12 +55,82 @@
                     android:text="@string/no_profile"
                     android:textSize="14sp" />
         </RelativeLayout>
-
-        <androidx.recyclerview.widget.RecyclerView
-                android:id="@+id/rv"
+        <LinearLayout
                 android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:layout_marginTop="70dp" />
+                android:layout_height="match_parent"
+                android:layout_marginTop="70dp"
+                android:orientation="vertical">
+
+            <RelativeLayout
+                    android:id="@+id/active"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_gravity="center_horizontal"
+                    android:visibility="visible">
+
+                <androidx.cardview.widget.CardView
+                        android:id="@+id/activeCard"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:layout_marginStart="8dp"
+                        android:layout_marginTop="8dp"
+                        android:layout_marginEnd="8dp"
+                        android:elevation="16dp"
+                        android:padding="5dp"
+                        app:cardCornerRadius="8dp">
+
+                    <RelativeLayout
+                            android:id="@+id/activeProfile"
+                            android:layout_width="match_parent"
+                            android:layout_height="wrap_content"
+                            android:padding="5dp">
+
+                        <ImageView
+                                android:id="@+id/vibIcon"
+                                android:layout_width="60dp"
+                                android:layout_height="50dp"
+                                android:layout_centerVertical="true"
+                                android:src="@drawable/vibration" />
+
+                        <TextView
+                                android:id="@+id/activeName"
+                                android:layout_width="wrap_content"
+                                android:layout_height="wrap_content"
+                                android:layout_marginStart="15dp"
+                                android:layout_marginTop="5dp"
+                                android:layout_toEndOf="@id/vibIcon"
+                                android:text="Currently Active Profile: "
+                                android:textColor="@color/textColor"
+                                android:textSize="18sp"
+                                android:transitionName="profileName" />
+
+                        <TextView
+                                android:id="@+id/end"
```

Comment:
Change the id to endTimeTxt and similarly add Txt at the end for textviews, img at the end for imageview, layout for cardview/Layouts.
Do this only in the views you have added in this PR.